frontend/src/Form/beta/JsonSchemaForm.tsx

```diff
@@ -59,16 +59,18 @@ function DescriptionFieldTemplate() {
 }
 
 function ObjectFieldTemplate(props: ObjectFieldTemplateProps) {
+  const { title, properties, description } = props
```

Comment:
Instead of doing this on a separate line we can just do the following on the line above: 
```
function ObjectFieldTemplate({ title, properties, description }: ObjectFieldTemplateProps) {
```